### PR TITLE
Using lazy option loading, returning skeleton and search summary to be dependent on filter loading state.

### PR DESCRIPTION
**Pull request by cbusmand2l**

### CODE REVIEW

File: components/d2l-quick-eval/d2l-quick-eval-activities.js [hunk 1]
```diff
@@ -124,7 +124,8 @@ class D2LQuickEvalActivities extends mixinBehaviors(
 			<div class="d2l-quick-eval-activity-list-modifiers">
 				<d2l-hm-filter
 					href="[[filterHref]]"
-					token="[[token]]">
+					token="[[token]]"
+					lazy-load-options>
```
Comment (cbusmand2l) [inline components/d2l-quick-eval/d2l-quick-eval-activities.js]: Adding this attribute (and its equivalent in submissions) is the added lazy behaviour. The rest is reversing the original lazy loading implementation as there would have been issues with the UI sometimes displaying inaccurate (late) information.

File: components/d2l-quick-eval/d2l-quick-eval-activities.js [hunk 2]
```diff
@@ -388,8 +388,8 @@ class D2LQuickEvalActivities extends mixinBehaviors(
 		return _data.length && !_showLoadingSkeleton;
 	}
 
-	_computeShowLoadingSkeleton(_loading, searchLoading) {
-		return _loading || searchLoading;
+	_computeShowLoadingSkeleton(_loading, filtersLoading, searchLoading) {
+		return _loading || filtersLoading || searchLoading;
```
Comment (maboelsoudd2l) [inline components/d2l-quick-eval/d2l-quick-eval-activities.js]: wasn't this the cause of an issue we had in https://github.com/BrightspaceHypermediaComponents/activities/pull/442?

Comment (AlexBedley) [reply components/d2l-quick-eval/d2l-quick-eval-activities.js]: yes but it's no longer an issue due to the changes that Carol made in hm-filter and we need to revert the change in #442

Comment (maboelsoudd2l) [reply components/d2l-quick-eval/d2l-quick-eval-activities.js]: cool just being thorough

**Pull request merged**